modules/dr-restoring-cluster-state.adoc

```diff
@@ -97,10 +97,51 @@ The output of this command should be empty. If it is not empty, wait a few minut
 $ sudo mv /var/lib/etcd/ /tmp
 ----
 
+.. If the `/etc/kubernetes/manifests/keepalived.yaml` file exists, follow these steps:
+
+... Move the `/etc/kubernetes/manifests/keepalived.yaml` file out of the kubelet manifest directory:
++
+[source,terminal]
+----
+$ sudo mv /etc/kubernetes/manifests/keepalived.yaml /tmp
+----
+
+... Verify that any containers managed by Keepalived are stopped:
++
+[source,terminal]
+----
+$ sudo crictl ps --name keepalived
+----
++
+The output of this command should be empty. If it is not empty, wait a few minutes and check again.
+
+... Check if the control plane has any Virtual IPs (VIPs) assigned to it:
++
+[source,terminal]
+----
+$ ip -o address | egrep '<api_vip>|<ingress_vip>'
+----
+
+... For each reported VIP, run the following command to remove it:
++
+[source,terminal]
+----
+$ sudo ip address del <reported_vip> dev <reported_vip_device>
+----
+
 .. Repeat this step on each of the other control plane hosts that is not the recovery host.
 
 . Access the recovery control plane host.
 
+. If Keepalived is in use, verify that the recovery control plane node owns the VIP:
```

Comment:
Reading the sentence is a bit stiled with "Keepalived", maybe convert it to an adjective and follow up with a concrete now such as "instance", "load balancer" and so on.

Reply:
Perhaps "If the Keepalived service is in use"?

Reply:
Ah, no, it's a daemon. Will update the wording accordingly. Thanks!